test/parser/open-api/data/oneOfResolver/outputs/double.json

```diff
@@ -0,0 +1,56 @@
+{
+  "oneOf": [
+    {
+      "type": "object",
+      "properties": {
+        "foo": {
+          "title": "Fooo",
+          "type": "string"
+        },
+        "bar": {
+          "title": "Bar"
+        }
+      },
+      "required": ["bar"]
+    },
+    {
+      "type": "object",
+      "properties": {
+        "foo": {
+          "title": "Fooo",
+          "type": "integer"
+        },
+        "bar": {
+          "title": "Bar"
+        }
+      },
+      "required": ["foo"]
+    },
+    {
+      "type": "object",
+      "properties": {
+        "foo": {
+          "title": "Fooo",
+          "type": "integer"
+        },
+        "bar": {
+          "title": "Bar"
+        }
+      },
+      "required": ["bar"]
+    },
+    {
+      "type": "object",
+      "properties": {
+        "foo": {
+          "title": "Fooo",
+          "type": "string"
+        },
+        "bar": {
+          "title": "Bar"
+        }
+      },
+      "required": ["foo"]
+    }
+  ]
+}
```

Comment:
I would reorder the elements in this file for better readability. Currently the order is as follows:
- string - bar
- integer - foo
- integer - bar
- string - foo